src/BlazorUI/Bit.BlazorUI/Components/Surfaces/Collapse/BitCollapse.scss

```
@@ -15,21 +23,125 @@
+.bit-col-cex {
+    opacity: 1;
+    grid-template-rows: 1fr;
 }
 
 .bit-col-cco {
     opacity: 0;
-    max-height: 0;
-    margin: spacing(-1.5) 0;
+    visibility: hidden;
+    grid-template-rows: 0fr;
+    // Opening shows the content at once and then grows the track; closing has to keep the content on the
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)